### DISCUSSION TITLE
taro nut ui 主题相关

### DISCUSSION THREAD

**Discussion by Arbureva** [NervJS/taro#12647] [Q & A]
目前看来 nutui 的绝大多数框架都没有设置颜色的选项，并且部分框架使用起来有卡顿现象，很难想象靠这样的表现taro能有这么多star，是否有什么性能优化方案和主题切换方案？你们的项目不会都是红色吧，我在使用日期选择器时就遇到了无法切换主题色的问题

**Comment by TheKonka** [comment]
https://github.com/jdf2e/nutui

可以到这边问问看，`nutui` 只是组件库，`taro` 是一套代码适配多端的框架
